src/database/DataTypes/RequisitionItem.js

```diff
@@ -243,7 +243,7 @@ RequisitionItem.schema = {
     dailyUsage: { type: 'double', optional: true },
     imprestQuantity: { type: 'double', optional: true },
     requiredQuantity: { type: 'double', optional: true },
-    suppliedQuantity: { type: 'double', optional: true },
+    suppliedQuantity: { type: 'double', default: 0 },
```

Comment:
Since customer requisitions come from sync, the default has been set in 4D - so, won't effect anything and makes things nicer